CMakeLists.txt

```diff
@@ -7,9 +7,10 @@ if (CMAKE_CUDA_COMPILER)
     enable_language(CUDA)
     set(CMAKE_CUDA_STANDARD 14)
     set(CMAKE_CUDA_STANDARD_REQUIRED ON)
-    set(CMAKE_CUDA_FLAGS "-arch=sm_35 -use_fast_math ${CMAKE_CUDA_FLAGS}")
+    set(CMAKE_CUDA_FLAGS "-use_fast_math ${CMAKE_CUDA_FLAGS}")
     set(CMAKE_CUDA_FLAGS_DEBUG "-G ${CMAKE_CUDA_FLAGS_DEBUG}")
     set(CMAKE_CUDA_SEPARABLE_COMPILATION ON)
+    set(CMAKE_CUDA_ARCHITECTURES 61 72 75)
```

Comment:
61 is for GTX1060; 75 is for RTX2080.

What's 72 for?

Consider adding 86 for 30 series GPU support: https://developer.nvidia.com/cuda-GPUs

Reply:
72 is Xavier. Added 86